src/styles/card.scss

```diff
@@ -0,0 +1,107 @@
+@import 'stars';
+@import 'variables';
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  width: 200px;
+  border: 1px solid $white-border;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  background: $white;
+  transition: transform 300ms;
+
+  &__img {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__discription {
+    margin-bottom: 4px;
+    color: $main-accent;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__discription:hover {
+    color: $secondary-discription;
+  }
+
+  &__product-code {
+    color: $secondary;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__rating {
+    align-items: center;
+    display: flex;
+    justify-content: space-between;
+
+    &-reviews {
+      align-items: center;
+      height: 16px;
+      color: $main-accent;
+      font-size: 10px;
+      line-height: 14px;
+    }
+  }
+
+  &__price {
+    text-align: center;
+    display: flex;
+    width: 100%;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__price-price {
+    color: $secondary;
+    font-size: 12px;
+    font-weight: 400;
+  }
+
+  &__price-num {
+    color: $main-accent;
+    text-align: right;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+  }
+
+  &__buy {
```

Comment:
The class &__buy is missing the display property. It should be set to 'flex' to use justify-content.